test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -509,19 +511,49 @@ void RandomCodeOptions::setWeight(eth::Instruction _opCode, int _weight)
 	setWeights();
 }
 
-void RandomCodeOptions::addAddress(Address const& _address)
+void RandomCodeOptions::addAddress(Address const& _address, AddressType _type)
 {
-	addressList.push_back(_address);
+	switch(_type)
+	{
+		case AddressType::CALLONLY:
+			callAddressList.push_back(_address);
+			break;
+		case AddressType::ALL:
+		case AddressType::ACCOUNT:
+			accountAddressList.push_back(_address);
+			callAddressList.push_back(_address);
```

Comment:
an address like 0x0000000...02  is a precompiled address that should be available for call only. there should be no account in pre state for that address